R/distanceMatrix.R

```diff
@@ -4,17 +4,19 @@
 #' @param t An optional list of the same size as \code{q} containing evaluation
 #'   grids for each QTS.
 #' @param labels A character vector specifying labels for each QTS.
-#' @param step_pattern The choice for the step pattern of the warping path. (default to \code{dtw::symmetric2},
-#' which is symmetric, normalizable with no local slope constraints. See \code{\link[dtw]{stepPattern}} for more information).
-#'
+#' @inheritParams DTW
+#' @param normalized A boolean specifyng whether to compute normalized distance
+#'   between QTS.
+#'   Please note that not all step patterns are normalizable
+#'   (default: \code{FALSE}).
 #' @return A \code{\link[stats]{dist}} object storing the distance matrix
 #'   between QTS in a sample via DTW.
 #' @export
 #'
 #' @examples
 #' # TO DO
 
-distDTW <- function (q, t = NULL, labels = NULL, step_pattern = dtw::symmetric2)
+distDTW <- function (q, t = NULL, labels = NULL, step_pattern = dtw::symmetric2, normalized = FALSE)
```

Comment:
What happens if I use `step_pattern = dtw::symmetric1, normalized = TRUE`?

Reply:
The function returns a matrix containing "NA" and a warning message to indicate that the chosen step pattern may not be normalizable

Reply:
So are the results reliable? Are all the functions that use this function handle this case smoothly?